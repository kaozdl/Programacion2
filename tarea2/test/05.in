insertar_lista (16,marzo) .
ordenar_lista
insertar_lista (0,anterior) (0,ultimo) (16,enero) .
ordenar_lista
insertar_lista (13,abril) (27,abril) .
ordenar_lista
Fin
