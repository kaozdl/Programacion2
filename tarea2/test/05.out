1>(0,primero)(16,marzo)
2>2
(0,primero)(16,marzo)
(0,primero)(16,marzo)
(0,primero)(16,marzo)
3>(0,primero)(0,primero)(16,marzo)(0,anterior)(0,ultimo)(16,enero)
4>6
(0,primero)(0,primero)(16,marzo)(0,anterior)(0,ultimo)(16,enero)
(0,primero)(0,primero)(16,marzo)
(0,primero)(0,primero)(0,anterior)(0,ultimo)(16,marzo)(16,enero)
(0,primero)(16,marzo)
5>(0,primero)(0,primero)(0,primero)(16,marzo)(0,anterior)(0,ultimo)(16,enero)(13,abril)(27,abril)
6>9
(0,primero)(0,primero)(0,primero)(16,marzo)(0,anterior)(0,ultimo)(16,enero)(13,abril)(27,abril)
(0,primero)(0,primero)(0,primero)(0,anterior)(0,ultimo)(13,abril)(16,marzo)(16,enero)(27,abril)
(0,primero)(13,abril)(16,marzo)(27,abril)
7>Fin.
